tests/test_feedexport.py

```diff
@@ -2398,7 +2398,7 @@ def test_export_items(self):
         yield self.assertExported(items, header, rows, settings=settings)
 
     def test_wrong_path(self):
-        """If path is without %(batch_time)s and %(batch_id) an exception must be raised"""
+        """If path is without %(batch_id) an exception must be raised"""
```

Comment:
This change does not seem necessary.

Reply:
Click on accept suggestion for a free merge!

```suggestion
        """If path is without %(batch_time)s and %(batch_id) an exception must be raised"""
```

Reply:
Just saw this suggestion now, but I also added commit that does the same thing